packages/backend/.env.example

```
@@ -10,3 +10,5 @@ NEXT_PUBLIC_WORKOS_REDIRECT_URI=http://localhost:3001/callback
 # AI
 OPENROUTER_API_KEY=or_
 SKETCHI_APP_URL=http://localhost:3001
+MODEL_NAME=google/gemini-3-flash-preview
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MODEL_NAME key should go before the OPENROUTER_API_KEY key

(UnorderedKey)